saleor/core/models.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+from django.db import models
+
+
+class BaseNote(models.Model):
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, blank=True, null=True,
+        on_delete=models.SET_NULL)
+    date = models.DateTimeField(db_index=True, auto_now_add=True)
+    content = models.TextField()
+    is_public = models.BooleanField(default=True)
+
+    class Meta:
+        abstract = True
+        ordering = ('date', )
```

Comment:
Default ordering is by pk, in this case, ordering by date added does the same thing, I suppose it's not needed

Reply:
We should also make sure that ordering is always stable, it's possible for two dates to be equal (even if unlikely) so if we really want to sort by date, I suggest that we sort by `('date', 'pk')`.

Reply:
Wa agreed that ordering by pk will be enough.